docs/ru/agg_functions/reference.md

```diff
@@ -227,82 +251,114 @@ quantileTiming(level)(x)
 
-quantileTDigest(level)(x)
--------------------------
+
+## quantileTDigest(level)(x)
 
 Вычисляет квантиль уровня level приближённо, с использованием алгоритма [t-digest](https://github.com/tdunning/t-digest/blob/master/docs/t-digest-paper/histo.pdf). Максимальная погрешность составляет 1%. Расход памяти на состояние пропорционален логарифму от количества переданных значений.
 
 Производительность функции ниже `quantile`, `quantileTiming`. По соотношению размера состояния и точности, функция существенно лучше, чем `quantile`.
 
 Результат зависит от порядка выполнения запроса, и является недетерминированным.
 
-median
-------
+
+## median
 
 Для всех quantile-функций, также присутствуют соответствующие median-функции: `median`, `medianDeterministic`, `medianTiming`, `medianTimingWeighted`, `medianExact`, `medianExactWeighted`, `medianTDigest`. Они являются синонимами и их поведение ничем не отличается.
 
-quantiles(level1, level2, ...)(x)
----------------------------------
+
+## quantiles(level1, level2, ...)(x)
 
 Для всех quantile-функций, также присутствуют соответствующие quantiles-функции: `quantiles`, `quantilesDeterministic`, `quantilesTiming`, `quantilesTimingWeighted`, `quantilesExact`, `quantilesExactWeighted`, `quantilesTDigest`. Эти функции за один проход вычисляют все квантили перечисленных уровней и возвращают массив вычисленных значений.
 
-varSamp(x)
-----------
+
+## varSamp(x)
 
 Вычисляет величину `Σ((x - x̅)2) / (n - 1)`, где n - размер выборки, x̅ - среднее значение x.
 
 Она представляет собой несмещённую оценку дисперсии случайной величины, если переданные в функцию значения являются выборкой этой случайной величины.
 
 Возвращает Float64. В случае, когда `n <= 1`, возвращается +∞.
 
-varPop(x)
----------
+## varPop(x)
 
 Вычисляет величину `Σ((x - x̅)2) / n`, где n - размер выборки, x̅ - среднее значение x.
 
 То есть, дисперсию для множества значений. Возвращает Float64.
 
-stddevSamp(x)
--------------
+## stddevSamp(x)
 
 Результат равен квадратному корню от `varSamp(x)`.
 
-stddevPop(x)
-------------
+
+## stddevPop(x)
 
 Результат равен квадратному корню от `varPop(x)`.
 
-covarSamp(x, y)
----------------
+
+## topK
+
+Возвращает массив наиболее часто встречающихся значений в указанном столбце. Результирующий массив упорядочен по убыванию частоты значения (не по самим значениям).
+
+```
+topK(N)(column)
+```
+
+Функция не дает гарантированного результата, при определенных условиях возможны ошибки и вернутся частые, но не наиболее частые значения.
+
+Рекомендуем использовать значения `N <= 10`, поскольку на небольших `N` функция выполняет задачу эффективнее, чем алгоритм, основанный на сортировке. Максимально возможное значение `N = 65536`.
+
+**Аргументы**
+- 'N' - Количество значений.
+- 'x' - Столбец.
+
+**Потребление памяти**
+ - `N * (the_size_of_the_key + 16)` байт для счетчиков.
+ - `48 * N` байт для отображения (map) альфа-значений.
```

Comment:
Useless and cryptic information.
The user cannot and should not understand, what are "counters" and "alpha values".

And this text doesn't provide information how to calculate memory usage.